flexmeasures/data/services/job_cache.py

```diff
@@ -0,0 +1,33 @@
+"""
+Logic around storing and retrieving jobs from redis cache.
+"""
+
+
+class JobCache:
+    def __init__(self, connection, queues):
+        self.connection = connection
+        self.queues = queues
+
+    def add(self, sensor_id, job_id):
```

Comment:
If we use assets as well (like you said), just the ID won't be enough (could lead to overlaps).

Maybe `f"sensor-{asset_or_sensor_id}"` or `f"asset-{asset_or_sensor_id}"`, depending on what type gets passed?